operator/e2e/e2e_test.go

```diff
@@ -45,7 +45,7 @@ func TestE2E(t *testing.T) {
 	b := make([]byte, 12)
 	_, _ = rand.Read(b) //nolint:staticcheck // Don't need crypto here.
 	tag := hex.EncodeToString(b)
-	projectimage := "pulumi/pulumi-kubernetes-operator-v2:" + tag
+	projectimage := "pulumi/pulumi-kubernetes-operator:" + tag
```

Comment:
I worry about folks currently using `pulumi/pulumi-kubernetes-operator:latest`. This has the potential to break people, or at least require some manual migration steps -- assuming we eventually publish v2 to latest.

Reply:
I sympathize. My plan is to NOT apply a suffix, but also NOT push to latest.